types/porty/index.d.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * The host to run on
+ * @default "0.0.0.0"
+ */
+export let HOST: string;
```

Comment:
None of these exported lets are documented in the README; are you sure they should be included?

Reply:
When I looked at the code I assumed this was some intended way of setting defaults, but if it's not mentioned in the README then I'll remove it from the definitions since it probably isn't really supposed to be exposed